WTWSMS/common/traits/WtWSMS_traits.txt

```diff
@@ -141,11 +141,14 @@ legendary_leader = {
 
 the_prophet = {
 	
-	muslim_opinion = 25
-	martial = 6
+	muslim_opinion = 100
+	martial = 12
```

Comment:
He's the Prophet of Islam. Are you seriously telling me that the Prophet of Islam shouldn't have maximum opinion with all Muslimsn while he is alive? Really?